source/custom/nidcpower_service.custom.cpp

```diff
@@ -1,5 +1,51 @@
 #include <nidcpower/nidcpower_service.h>
 
+#include <stdexcept>
+
 namespace nidcpower_grpc {
 
+struct DriverErrorException : std::runtime_error {
+  DriverErrorException(int status) : std::runtime_error("") { status_ = status; }
```

Comment:
Set `status_` in the initializer section:
```cpp
DriverErrorException(int status) : std::runtime_error(""), status_(status) { }
```

Reply:
I have used this initializer list style only.

Reply:
Looks good. Marking resolved.